pkg/core/lifecycle.go

```diff
@@ -389,6 +389,6 @@ func (pl *Pipeline) getOauthSecret(repoID, gitProvider string) (*Oauth, error) {
 		pl.Logger.Errorf("error while unmarshaling json to oauth for RepoID %s : %s", repoID, err)
```

Comment:
Did you do the corresponding changes in neuron token API?

Reply:
yes made the change

Reply:
If we have removed the API in neuron, we should also remove this function and directly fetch secrets from secret parser